dedibox/vps/how-to/manage-vps.mdx

```diff
@@ -0,0 +1,67 @@
+---
+meta:
+  title: How to manage a Dedibox VPS
+  description: This page explains how to manage a Dedibox VPS
+content:
+  h1: How to manage a Dedibox VPS
+  paragraph: This page explains how to manage a Dedibox VPS
+tags: vps dedibox manage
+dates:
+  validation: 2024-05-15
+  posted: 2024-05-15
+categories:
+  - dedibox-vps
+---
+
+You can manage your Dedibox VPS directly from the Dedibox console.
+
+<Macro id="requirements" />
+
+- A Dedibox account logged into the [console](https://console.online.net)
+- [Created a Dedibox VPS](/dedibox/vps/how-to/create-vps/)
+
+## How to manage a Dedibox VPS from its information dashboard
+
+1. Open the Dedibox console home page and click **Dedibox VPS** in the navigation at the top of the page. This will bring up a list of your virtual machines. 
```

Comment:
```suggestion
1. Log in to your Dedibox console account and click **Dedibox VPS** in the navigation at the top of the page. This will bring up a list of your Dedibox VPS. 
```